auth-service/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
-val ktorVersion = "2.3.9"
-val kotlinVersion = "1.9.23"
+val ktorVersion = "2.3.11"
+val kotlinVersion = "2.0.0"
```

Comment:
Let's not jump major Kotlin versions yet, can you raise a ticket for it. Make sure the versions just below in the `plugins` block match these

Reply:
added versions to block below and ticket raised